test/e2e/framework/internal/junit/junit.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
```

Comment:
Does this file is actually tested anywhere? if not, why not leave it until the UT is possible?

Reply:
This is an oversight: output.go was meant to call it instead of using a copy. Fixed.